project/cmake/kodi-config.cmake.in

```diff
@@ -16,7 +16,7 @@ IF(NOT WIN32)
   SET(CMAKE_CXX_FLAGS "$ENV{CXXFLAGS} @CXX11_SWITCH@")
 ENDIF()
 LIST(APPEND CMAKE_MODULE_PATH @APP_LIB_DIR@)
-ADD_DEFINITIONS(@ARCH_DEFINES@ -DBUILD_KODI_ADDON)
+ADD_DEFINITIONS(@APP_ARCH_DEFINES@ -DBUILD_KODI_ADDON)
```

Comment:
this probably breaks the cmake addon build system, need some changes in prepare-env.cmake